publications.json

```diff
@@ -7,15 +7,47 @@
          "rssURL":"http://theadvocatecornell.com/feed/",
          "name":"The Advocate",
          "slug":"advocate",
-         "websiteURL":"http://theadvocatecornell.com/"
+         "socialURLs":[
+            {
+               "social":"insta",
+               "URL":"https://www.instagram.com/theadvocateatcornelluniversity"
+            },
+            {
+               "social":"facebook",
+               "URL":"https://www.facebook.com/The-Advocate-at-Cornell-University-109632540936869"
+            }
+         ],
+         "websiteURL":"http://theadvocatecornell.com"
       },
       {
          "bio":"Student-run entertainment and media organization dedicated exclusively to Cornell University Athletics.",
          "bioShort":"Dedicated exclusively to Cornell Athletics",
          "rssName":"Big Red Sports Network",
-         "rssURL":"https://www.bigredsportsnetwork.org/feed/",
+         "rssURL":"https://www.bigredsportsnetwork.org/feed",
          "name":"Big Red Sports Network",
          "slug":"brsn",
+         "socialURLs":[
+            {
+               "social":"insta",
+               "URL":"https://www.instagram.com/cornellbrsn"
+            },
+            {
+               "social":"facebook",
+               "URL":"https://www.facebook.com/CornellBRSN"
+            },
+            {
+               "social":"youtube",
+               "URL":"https://www.youtube.com/channel/UCXl9QF-FaKPR3ldQ9zkG0jg/videos"
+            },
+            {
+               "social":"linkedin",
+               "URL":"https://www.linkedin.com/company/bigredsportsnetwork"
+            },
+            {
+               "social":"twitter",
+               "URL":"https://twitter.com/CornellBRSN"
+            }
+         ],
          "websiteURL":"https://www.bigredsportsnetwork.org/"
```

Comment:
is there a reason why this has trailing slash and the others don't

Reply:
That was def an accident

Reply:
will fix